skyportal/handlers/api/analysis.py

```diff
@@ -1310,3 +1346,196 @@ async def get(
                 )
 
             return self.error("No data found for this Analysis.", status=404)
+
+
+class AnalysisUploadHandler(BaseHandler):
```

Comment:
Should this be somehow named UploadOnly or similar? To differentiate it more somehow

Reply:
Good idea.